File: JetpckStudy/app/src/main/java/com/daysnap/jetpckstudy/MainActivity.java
package com.daysnap.jetpckstudy;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btn_lifecycle).setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        int id = v.getId();
        if (id == R.id.btn_lifecycle) {
            Intent intent = new Intent(this, LifecycleActivity.class);
            startActivity(intent);
        }
    }
}

File: TaoBaoUnion/app/src/main/java/com/owulia/taobaounion/MainActivity.kt
package com.owulia.taobaounion

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.MenuItem
import android.view.View
import com.google.android.material.bottomnavigation.BottomNavigationView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    val tag = "MainActivity"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initView()
        initListener()
    }

    private fun initView () {

    }

    private fun initListener () {
        navBar.setOnNavigationItemSelectedListener{
            Log.d(tag, "title => ${it.title}")
            false
        }
    }

}
